.github/workflows/crowdin-build-translations.yml

```diff
@@ -0,0 +1,21 @@
+name: Crowdin Build Translations
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 5 * * 1,4'
+
+jobs:
+  build:
+    name: Build Translations
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Java
+      uses: actions/setup-java@v1
+      with:
+        java-version: 8
```

Comment:
Build with 8 but download with 11?

Reply:
I guess that makes sense as we're still building 8 but may as well use 11 for non-build stuff. Less to change later.

Reply:
Build uses just this repo, download/copy uses all of them, picked 11 for the later just to be safe (although 8 should work just fine).